src/core_modules/capture-core/components/DataEntries/common/useMetadataForRegistrationForm/buildFunctions/buildEnrollmentForm.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import { EnrollmentFactory } from '../../../../../metaDataMemoryStoreBuilders/programs/factory/enrollment';
+import type {
+    OptionSet,
+    TrackedEntityAttribute,
+} from '../../../../WidgetProfile/DataEntry/FormFoundation/types';
```

Comment:
Hey,
In the future, `WidgetProfile` will belong to an external component library. Therefore, I think we should avoid imports that use its nested folder structure. I propose to change `'../../../../WidgetProfile/DataEntry/FormFoundation/types'`; to `'../../../../WidgetProfile'` and make the necessary changes inside the `WidgetProfile` to export the types.

PS: Maybe it makes sense to create the needed types somewhere in the `useMetadataForRegistrationForm` folder instead and not import them from the self-contained  `WidgetProfile`. WDYT?
Thanks!